ehr_billing/src/org/labkey/ehr_billing/EHR_BillingModule.java

```diff
@@ -64,7 +64,7 @@ public String getName()
     @Override
     public @Nullable Double getSchemaVersion()
     {
-        return 21.000;
+        return 21.001;
```

Comment:
This is the right approach normally, but in order to make the merging forward easier, lets use 23.001 here instead of 21.001